HW-27/hw27.js

```diff
@@ -0,0 +1,53 @@
+// 1. Додати на раніше створену сторінку кнопки “Додати в друзі”, “Написати повідомлення”, “Запропонувати роботу”.
+let sendMessageBtn = document.createElement("button");
+sendMessageBtn.innerHTML = "Написати повідомлення";
+sendMessageBtn.style.background = "purple"
+sendMessageBtn.addEventListener("click", function () {
+    alert("Напишіть своє повідомлення тут.");
+});
+document.body.appendChild(sendMessageBtn);
+
+
+let jobOfferBtn = document.createElement("button");
+jobOfferBtn.innerHTML = "Запропонувати роботу";
+jobOfferBtn.style.background = "purple"
+jobOfferBtn.addEventListener("click", function () {
+    alert("Відкривається форма для заповнення пропозиції роботи.");
+});
+document.body.appendChild(jobOfferBtn);
+
+// 2. Додати окремий елемент, що має відображати кількість друзів. Ця кількість має ініціалізуватися як рандомне число при завантаженні сторінки, а при натисканні на "Додати в друзі" збільшуватися на 1.
+
+var friendCount = Math.floor(Math.random() * 100) + 1;
+
+var addFriendBtn = document.getElementById("addFriendBtn");
+var friendCountSpan = document.getElementById("friendCount");
+
+friendCountSpan.innerHTML = friendCount + " друзів";
+
```

Comment:
The button should become unclickable (disabled) after the first click.

Reply:
And the text should be changed according to the task

Reply:
Add some changes on js file.